cli/plugin/plugin.go

```diff
@@ -25,7 +25,7 @@ import (
 	"github.com/buildbuddy-io/buildbuddy/server/util/git"
 	"github.com/buildbuddy-io/buildbuddy/server/util/status"
 	"github.com/creack/pty"
-	"gopkg.in/yaml.v2"
+	yaml "gopkg.in/yaml.v2"
```

Comment:
hmm, weird that this previously built? anyways, mind throwing a newline above so the aliased imports are separate from the unaliased ones?

Reply:
I think it's because my ide automatically removes unused imports and the parsing logic wasn't processing correctly that this was used, so it kept trying to remove it